src/Query/Wildcard.php

```diff
@@ -67,4 +67,15 @@ public function setRewrite(string $rewriteMode): self
 
         return $this;
     }
+
+    /**
+     * @see https://www.elastic.co/guide/en/elasticsearch/reference/7.10/query-dsl-wildcard-query.html#wildcard-query-field-params
+     */
```

Comment:
Should we add a note here, stating that this is only working for ES >= v7.10?

Reply:
++, that would be a nice indicator.

Reply:
Yes that will be the same for #1874 and setFixedInterval() and setCalendarInterval() but for ES 7.2